sdk/formrecognizer/Azure.AI.FormRecognizer/src/GetModelOptions.cs

```diff
@@ -1,12 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Azure.Core;
-
-namespace Azure.AI.FormRecognizer.Models
-{
-    [CodeGenModel("Enum0")]
-    internal partial struct GetModelOptions
```

Comment:
Removed this as it was not being used. Tried rerunning CodeGen after removal and got no changes.